### DISCUSSION TITLE
Selecting all items on relation, prevents them to show on drawer

### DISCUSSION THREAD

**Discussion by joselcvarela** [directus/directus#8032] [Feature Requests]
![image](https://user-images.githubusercontent.com/14039341/133279550-1d61bff8-dd52-4639-b5a0-f6fe12e5de38.png)

My first thought was my items disappearead.
What do you think?

**Comment by rijkvanzanten** [comment]
That works for me 👍🏻

**Reply by benhaynes** [reply]
I personally like the idea of showing all items, but with the selected items... selected. :)

**Reply by rijkvanzanten** [reply]
This was our way before to ensure people don't select the same item twice in the drawer

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
